app/models/concerns/resultable.rb

```diff
@@ -173,4 +172,75 @@ def to_s(field)
   def hlp
     ActionController::Base.helpers
   end
+
+  def wca_id
+    @personId || self.personId
```

Comment:
Now that `light_result` is gone, I think you don't need `@personId` anymore. You need to confirm this with a little bit of CTRL+F, but if I'm right then this should boil down to be a simple `alias_attribute`.